frontends/infinite-corridor/package.json

```diff
@@ -20,11 +23,14 @@
     "@types/react-router": "^5.1.18",
     "@types/react-router-dom": "^5.3.3",
     "axios": "^0.27.2",
+    "css-loader": "^6.7.1",
     "lodash": "^4.17.21",
+    "mini-css-extract-plugin": "^2.6.1",
```

Comment:
We were actually already using this in infinite-corridor. We were using version `0.5.0` (4 years old). production build was failing after I added ckeditor, and upgrading mini-css-extract-plugin fixed the issue.

(Open-discussions is still on 0.5.0.)